app/controllers/dashboard/projects_controller.rb

```diff
@@ -1,5 +1,22 @@
 class Dashboard::ProjectsController < Dashboard::BaseController
   inherit_resources
+  before_action :set_skills
```

Comment:
Why does this need to be here and in the superclass and in the base class?  It should only be needed in one class.